integration-tests/src/test/java/oracle/weblogic/kubernetes/ItCrossDomainTransaction.java

```diff
@@ -232,8 +233,12 @@ private static void addToPropertyFile(String propFileName, String domainNamespac
   private static void buildApplicationsAndDomains() {
 
     //build application archive
+    // Path targetDir = Paths.get(WORK_DIR, this.getClass().getSimpleName());
```

Comment:
remove commented code

Reply:
I leave it for suggestion, since I got a compilation error. Will remove it once we finalize the solution to get the class name.

Reply:
Removed the comments.